dart_io_mini_samples/example/websockets/using_websockets_server.dart

```diff
@@ -6,24 +6,18 @@
 /// `WebSocketTransformer.upgrade()`.
 
 import 'dart:io';
-import 'dart:async';
 
 handleMsg(msg) {
   print('Message received: $msg');
 }
 
-main() {
-  runZoned(() {
-    HttpServer.bind('127.0.0.1', 4040).then((server) {
-      server.listen((HttpRequest req) {
-        if (req.uri.path == '/ws') {
-          // Upgrade a HttpRequest to a WebSocket connection.
-          WebSocketTransformer.upgrade(req).then((socket) {
-            socket.listen(handleMsg);
-          });
-        }
-      });
-    });
-  },
-  onError: (e) => print("An error occurred."));
-}
\ No newline at end of file
+main() async {
+  var server = await HttpServer.bind('127.0.0.1', 4040);
```

Comment:
Should we have a try-catch around this to handle bind errors (or whatever other errors might occur)?